src/Components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const ErrorPage = ({ error }) => {
+  let errorMessage = 'An error occurred.';
+
+  if (error) {
+    if (error.status === 404) {
+      errorMessage = 'Page not found.';
+    } else if (error.status === 500) {
+      errorMessage = 'Internal server error.';
+    } else if (error.message) {
+      errorMessage = error.message;
+    }
+  }
```

Comment:
I'm wondering if we can make the error message a little friendlier for the 5XX errors & also potentially handle ALL 5xx errors, something like: 

if (error.status >= 500 && < 600) ... that way ANY server errors are handled?